xbmc/peripherals/Peripherals.cpp

```diff
@@ -63,6 +63,8 @@
 #include "guilib/LocalizeStrings.h"
 #endif
 
+#define HOLD_THRESHOLD 250
```

Comment:
`HOLD_THRESHOLD_MS`. 250 ms is relatively show. What duration is used for keyboard hold mod? Joysticks are very responsive input devices, so I made buttons configurable: https://github.com/xbmc/xbmc/blob/master/system/keymaps/joystick.xml#L76

Reply:
I used the same value that was used for keyboard (input/KeyboardStat.cpp). But honestly this was just to test it. Test cycle is 48 hours!!!

Could make it configurable if we choose too assuming it works.

Reply:
I wouldn't say most TV remotes are responsive though.